src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,435 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload tests running identical queries as tsbs_query in a Genny workload. During
+  development we noticed significant performance differences between Genny and tsbs
+  workloads running very similar queries. This workload is being used for development
+  to investigate these runtime differences. The data is preloaded by the dsi configuration.
+  See 'configurations/test_control/test_control.tsbs_query.yml' for all the details. There are 20736000 
+  documents in the collection.
+
+Keywords:
+  - timeseries
+  - aggregate
+  - group
+  - sort
+
+GlobalDefaults:
+  Database: &database cpu
+  Collection: &collection point_data
+  maxPhases: &maxPhases 12
+  Repeat: &repeat 10
+
+# Create variables to mimic the randomness of tsbs_generate_queries.
+#  The min and max time of the pre-loaded tsbs dataset.
+MinDate:
+  &MinDate {
+    ^RandomDate: { min: "2016-01-01T00:00:00Z", max: "2016-01-24T23:59:50Z" },
+  }
+
+MaxDate:
+  &MaxDate { ^RandomDate: { min: *MinDate, max: "2016-01-24T23:59:50Z" } }
+
+HostName1: &HostName1
```

Comment:
Please add comments along the lines of:
```
# Generates a single string like "host_42", with random integer in range [0,99].
```
Same for HostName8 (think about the future readers of this file who have _no prior_ knowledge of the TSBS benchmarks).